apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -42,6 +43,7 @@ public class TablesClient {
   private final TableApi tableApi;
   private final DatabaseApi databaseApi;
   private final DatabaseTableFilter databaseFilter;
+  private final int cutOffHour;
```

Comment:
Does it belong to DatabaseTableFilter object? We could add apply method for GetTableResponseBody?

Reply:
IMO, it would be good to have a boundary for adding checks in `DatabaseTableFilter`. Preferably keep it limited to ops which are used to filter tablesList in JobsScheduler. 
Also, I followed it on the pattern of `isPrimaryTable` check which is used to preempt canRunX job.